README.md

```diff
@@ -78,6 +77,10 @@ path/to/patchConfig.xml
 # Attention: num should be greater than 1, because if num is 1, no report will be created.
 4
 ```
+then, add Environment variables:
+```bash
+checkstyle.pathfilter.patch.from.env=/path/to/patch-filters/DiffReport/patch.txt
```

Comment:
Use `checkstyle.patchfilter.patch` for env variable name. Check my PR - https://github.com/checkstyle/contribution/pull/491, same names should be used in the current PR.

Reply:
done